src/Services/Volume-control.vala

```diff
@@ -369,9 +367,12 @@ public class Sound.Services.VolumeControlPulse : VolumeControl {
                         break;
                 }
                 /* Only switch the active sink input in case a phone one is not active */
-                if (_active_sink_input == -1 ||
-                        _sink_input_hash.get (_active_sink_input) != _objp_role_phone)
+                if (
+                    _active_sink_input == -1
+                    || _sink_input_hash.get (_active_sink_input) != _objp_role_phone
```

Comment:
I'm personally not a fan of this

Reply:
Would you prefer to only split to a newline at the ||?

Reply:
whoops, regardless, this can be one-lined. my bad!